Auth0/SafariWebAuth.swift

```diff
@@ -43,6 +43,17 @@ class SafariWebAuth: WebAuth {
     var nonce: String?
     var leeway: Int = 60 * 1000 // Default leeway is 60 seconds
     var maxAge: Int?
+
+    lazy var redirectURL: URL? = {
```

Comment:
lazy means it will only get populated when required, right? e.g., if the user has set up a custom redirectURL value before launching the authentication, this lambda won't ever get called. Am I right?

Reply:
Exactly. From the [Swift docs](https://docs.swift.org/swift-book/LanguageGuide/Properties.html):

> A lazy stored property is a property whose initial value is not calculated until the first time it is used. You indicate a lazy stored property by writing the lazy modifier before its declaration.